app/assets/stylesheets/sass/components/_appointments.scss

```diff
@@ -0,0 +1,76 @@
+.appointment__container {
+  $base-border-radius: 3px !default;
+  $base-spacing: 1.5em !default;
+  $action-color: #477DCA !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;
+  $appointment-gutter: 1.4em;
+  $appointment-image-padding: 0.7em;
+  $appointment-image-width: 6em;
+  $appointment-color: $base-font-color;
+  // $appointment-background: lighten($secondary-color, 15%);
+  $appointment-background: $light-gray;
+  $appointment-detail-color: transparentize($appointment-color, 0.5);
+  $appointment-image-vert-alignment: top;
+
+  border-bottom: 1px solid transparentize($appointment-color, 0.9);
+  display: table;
+  margin-bottom: $base-spacing;
+  padding-bottom: 1em;
+  width: 100%;
+
+  .appointment__date,
+  .appointment__content {
+    display: table-cell;
+    vertical-align: $appointment-image-vert-alignment;
+  }
+
+  .appointment__date {
+    padding-right: $appointment-gutter;
```

Comment:
Properties should be ordered background, border-radius, display, height, max-width, padding, padding-right, text-align, width